### ISSUE TITLE
Request: recommended workflow for new devs

### ISSUE THREAD

**Issue by tmcclintock** [RPTools/maptool#1376]
**Is your feature request related to a problem? Please describe.**
I would like to contribute to MapTool (working from a Mac). The README has instructions up though actually building MapTool, but no instructions no how to, e.g., run unit tests or generally what a best-practice workflow would entail.

**Describe the solution you'd like**
A list of recommended steps such as:

1. Edit source files found [here].
2. Put new tests in [this] location.
3. Follow steps XYZ to rebuild the dev version of MapTool.
4. Run the entire tests using ABC.
5. Any extra instructions for issuing a PR.

**Describe alternatives you've considered**
There is no clear entry point for new contributors, but the 'up for grabs' issue label is appreciated.

**Additional context**
Looking forward to making MapTool even better!

**Comment by JamzTheMan** [comment]
Yea, we've been meaning to create a CONTRIBUTE.md file for awhile. I bring it up on our Discord server.

**Comment by tmcclintock** [comment]
Awesome, thank you. Would the discord be a good place to seek guidance on getting started in the meantime?

**Comment by JamzTheMan** [comment]
Absolutely. We're pretty active there and have a group of channels dedicated to contributing.

https://discord.gg/gevEtpC
